### PR TITLE
refork and add some models

### PR DESCRIPTION
**Pull request by Kevin-Zh-CS**

### CODE REVIEW

File: model/department.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package model
+
+import (
+	"github.com/go-gorm/gorm"
+)
+
+type Department struct {
+	gorm.Model
+	DepartmentName string 		//科室名
```
Comment (AsterNighT) [inline model/department.go:9]: Use english pls

File: model/department.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	Detail string				//科室简介
+	Capacity int				//科室容量，
+	Remain int
+	Waiting int //Waiting = Capacity - Remain
```
Comment (AsterNighT) [inline model/department.go:13]: So why bother stroing it if you can get it by simple calculation?

File: model/doctor.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package model
+
+import (
+	"github.com/go-gorm/gorm"
+)
+
+type Doctor struct {
+	gorm.Model
```
Comment (AsterNighT) [inline model/doctor.go:8]: Don't use this, you'll struggle with swaggo

**Review by AsterNighT — CHANGES_REQUESTED** [review]
You are going too fast on this, pls do submit your opinions under other groups' pr, rather than directly create a new one to overwrite everything.

**Comment by Kevin-Zh-CS** [comment]
I created some models based on the functions of our group, but some attributes may need to be added to other groups. For example, the doctor model, Our group only uses the departmentId attribute, but orther groups may need to fill in attributes such as account and password. So maybe we need discussion. @AsterNighT @PAN-Ziyue @FUTURETECH6 @Frank-Skywalker

File: model/doctor.go (hunk 3 above)
Comment (Kevin-Zh-CS) [reply model/doctor.go:8]: But our ORM framework can automatically introduce basic CRUD through this. I am doing GO project for the first time and I am not very familiar with it. Sorry.

**Comment by Kevin-Zh-CS** [comment]
> You are going too fast on this, pls do submit your opinions under other groups' pr, rather than directly create a new one to overwrite everything.

File: model/doctor.go (hunk 3 above)
Comment (AsterNighT) [reply model/doctor.go:8]: There's no need to be sorry, and I would like to here about the `automatically introduce basic CRUD` part. I don't read any of that, can you provide some information?

**Pull request closed without merging**